### DISCUSSION TITLE
Accessibility status

### DISCUSSION THREAD

**Discussion by pbb72** [quasarframework/quasar#12441] [General - Components / Directives / etc] [answered]
We are required by law (as I think is the case in many countries) to make our tools accessible. That is, confirm to [WCAG](https://www.w3.org/WAI/WCAG21/quickref/).

My first impression from the Quasar guide were not very good. Tabbing through the page showed that most focusable elements have their focus indication actively disabled (e.g. `.doc-link { outline: 0; }`), and a quick test of the Date Picker and Splitter components showed they are not focusable at all.

Is it relatively easy to use Quasar components in an accessible way, or would this be a major headache? The documentation states Quasar has ["Accessibility features"](https://quasar.dev/introduction-to-quasar#best-practices-integrated-by-default) (which of course could mean anything), but seeing accessibility features actually disabled makes me fear the worst.

**Comment by hawkeye64** [comment, marked as answer]
We have been working hard to get this done. If you see deficiencies, please raise an issue. If you have a fix, please PR and reference the issue. Thanks.

**Comment by ryentzer** [comment]
@pbb72 What do you use to test for accessibility? I'm running into similar issues with components getting flagged with accessibility errors. I mostly use WebAim's Wave extension, but follow up with Axe Devtools and the built in accessibility devtool in Firefox.

@hawkeye64 Is there a branch specific to addressing WCAG? Should we create an issue for each specific error? For instance the q-table is getting flagged for multiple errors. Thanks.

**Reply by pbb72** [reply]
@ryentzer Wave, Axe and the Chrome & Firefox devtools are good starting points. SiteImprove also has some good tools (https://www.siteimprove.com/integrations/browser-extensions/). But many aspects are not (easily) machine-testable, and require manual testing and common sense.

**Reply by ryentzer** [reply]
Thank you @pbb72! I hadn't heard of SiteImprove, checking them out now.
